### DISCUSSION TITLE
请问群被封禁是否会产生通知事件？

### DISCUSSION THREAD

**Discussion by SK-415** [Mrs4s/go-cqhttp#1623] [Q&A] [answered]
我目前想实现一个功能：当 Bot QQ 号 `主动退群`、`被踢出群`、`群解散` 和 `群被封禁` 时，自动删除对应群的配置文件。

但是 [go-cqhttp 文档](https://docs.go-cqhttp.org/event/#%E7%BE%A4%E6%88%90%E5%91%98%E5%87%8F%E5%B0%91) 和 [onebot 文档](https://github.com/botuniverse/onebot-11/blob/master/event/notice.md#%E7%BE%A4%E6%88%90%E5%91%98%E5%87%8F%E5%B0%91) 中的 `群成员减少事件` 都没有提及 `群解散` 和 `群被封禁` 时的表现。

经过测试 `群解散` 的表现与 `主动退群` 一致，`群被封禁` 缺乏条件无法测试。

请问 `群被封禁` go-cqhttp 是否会产生一个通知事件？类型字段的名称和值是什么？还是说不会产生事件，只能主动检查群是否存在。

**Comment by Ink-33** [comment, marked as answer]
应该是没有这个事件的，而且可能也没有做群被封的数据包解析
